cmd/notary/tuf.go

```diff
@@ -889,7 +889,13 @@ func tokenAuth(trustServerURL string, baseTransport *http.Transport, gun data.GU
 	if endpoint.Scheme == "" {
 		return nil, fmt.Errorf("Trust server url has to be in the form of http(s)://URL:PORT. Got: %s", trustServerURL)
 	}
-	subPath, err := url.Parse("v2/")
+	var p string
+	if strings.HasSuffix(endpoint.Path, "/") {
```

Comment:
Non-blocking:  If you wouldn't mind, could this URL validation be moved to a utility function in `storage/httpstore.go` so that it could be shared between this and the `HTTPStore`?  It'd be useful for library users of the notary client as well.

It would also make it easier to test, since you can just test that function alone (because the second thing I was going to ask is a test for this particular validation withsubpaths :))